.gitlab/pipeline/jobs/unit-php8.1-v11.yml

```diff
@@ -0,0 +1,10 @@
+unit-php8.1-v11:
+  extends: .default
+  image: php:8.1
+  stage: test
+  needs:
+    - build-composer-dependencies
+    - php-lint-php8.1
+  script:
+    - composer require --no-progress typo3/minimal:"^11.5"
```

Comment:
Should this not be `typo3/cms-core` instead ? (#530 for example) ? Not rebased correctly ? Or mabye slipped through there.